templates/layout.html

```diff
@@ -6,8 +6,6 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>{% block title %}{% endblock %}</title>
-    <link rel="stylesheet" type="text/css" href="../static/css/bootstrap.min.css">

```

Comment:
I plan to SASS integration which will make one compressed CSS file. The spectrum changes can be a part of it.